modules/@apostrophecms/doc-type/index.js

```diff
@@ -520,11 +520,17 @@ module.exports = {
 
       async convert(req, input, doc, options = {
         presentFieldsOnly: false,
-        copyingId: false
+        copyingId: false,
+        isPage: false
```

Comment:
Eh? I considered doing a `__meta.chain` thing to see if it was a page, but this is only called twice for pages, I think.

Reply:
A page type will never stop being a page type, even if the particular page type changes.

So we can write `self.apos.instanceOf(module, '@apostrophecms/piece-type')` here, and avoid requiring the level above to inject something we should already know as a manager of a specific type.

Reply:
Never mind, better idea below.